src/undocumented.md

```diff
@@ -9,24 +9,20 @@ to shrink!
 - [`libstd` facade]
 - [Trait reform] – some partial documentation exists (the use of `Self`), but
   not for everything: e.g. coherence and orphan rules.
-- [Attributes on `match` arms] – the underlying idea is documented in the
-  [Attributes] section, but the applicability to internal items is never
-  specified.
 - [Flexible target specification] - Some---but not all---flags are documented
   in [Conditional compilation]
-- [Require parentheses for chained comparisons]
+- Require parentheses for chained comparisons
```

Comment:
This is documented [here](https://doc.rust-lang.org/nightly/reference/expressions/operator-expr.html#comparison-operators)